public/app/plugins/datasource/prometheus/partials/config.html

```diff
@@ -1,3 +1,14 @@
 <datasource-http-settings current="ctrl.current" suggest-url="http://localhost:9090">
 </datasource-http-settings>
 
+<h3 class="page-heading">Prometheus details</h3>
+
+<div class="gf-form-group max-width-30">
+	<div class="gf-form">
+		<label class="gf-form-label width-13">Label Keys</label>
+		<input type="text" class="gf-form-input max-width-18" ng-model='ctrl.current.jsonData.labelKeys' placeholder="job,instance"></input>
```

Comment:
I change to gather label keys from query response.
User don't need to set label key manually.